src/main/java/no/entur/uttu/export/netex/producer/NetexObjectFactory.java

```diff
@@ -324,14 +318,19 @@ public ServiceFrame createCommonServiceFrame(
       }
     }
 
+    ServiceLinksInFrame_RelStructure serviceLinksInFrame_relStructure = objectFactory
+            .createServiceLinksInFrame_RelStructure()
+                    .withServiceLink(serviceLinks);
+
     return createServiceFrame(context)
       .withRoutePoints(routePointStruct)
       .withScheduledStopPoints(scheduledStopPointsStruct)
       .withStopAssignments(stopAssignmentsStruct)
       .withNetwork(network)
       .withAdditionalNetworks(additionalNetworks)
       .withNotices(noticesInFrame_relStructure)
-      .withDestinationDisplays(destinationDisplaysInFrame_relStructure);
+      .withDestinationDisplays(destinationDisplaysInFrame_relStructure)
+      .withServiceLinks(serviceLinks.isEmpty() ? null : serviceLinksInFrame_relStructure);
```

Comment:
This null thing was important, for some reason having <serviceLinks> with nothing inside would throw an exception